// Copyright 2013 The Chromium Authors
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

syntax = "proto2";

option optimize_for = LITE_RUNTIME;

package enterprise_management;

option go_package="chromium/policy/enterprise_management_proto";

// Describes how to retrieve policy data for a particular extension. The
// extension ID is in the |settings_entity_id| field of the PolicyData message
// that contains the ExternalPolicyData in its |policy_value| field.
message ExternalPolicyData {
  reserved 3;

  // A URL where the policy data can be downloaded from.
  optional string download_url = 1;

  // SHA-256 hash of the data at |download_url|. This is used to verify the
  // integrity of the data, and to detect updates on the client side: the client
  // downloads the data when its local hash does not match |secure_hash|.
  optional bytes secure_hash = 2;
}
